Packs/ReversingLabs_A1000/Integrations/ReversingLabsA1000v2/ReversingLabsA1000v2.py

```diff
@@ -416,7 +417,11 @@ def advanced_search(a1000):
         limit = 5000
 
     try:
-        result_list = a1000.advanced_search_v2_aggregated(query_string=query, max_results=limit)
+        result_list = a1000.advanced_search_v2_aggregated(
```

Comment:
`ticloud` is optional, please check if the user set it before pass it to `advanced_search_v2_aggregated`

Reply:
The `ticloud` argument is optional but I don't have to check if the user set it because its default value is set to `False` in the YML file.
Also, `advanced_search_v2_aggregated()` is a method froum our ReversingLabs SDK (https://github.com/reversinglabs/reversinglabs-sdk-py3/blob/main/ReversingLabs/SDK/a1000.py#L2004) and also there it defaults to `False` so there's no need for checking here.